ws-nextjs-app/pages/[service]/live/[id]/live.stories.tsx

```diff
@@ -8,6 +8,7 @@ const mockPageData = {
   ...liveFixture.data,
   liveTextStream: {
     content: postFixture,
+    contributors: 'Not a random dude',
```

Comment:
Not sure if this is okay to put in as the fixture data? Figured it shouldnt be null at least as it needs to be viewed on storybook. Does there need to be an option in storybook to view it with and without contributors?

Reply:
I think its ok for now. We don't have individual stories for the different 'states' of the Live page yet, so we can expand on them in the future once we are happy with the first iteration.